### ISSUE TITLE
Payment Item Quantity can not be calculated

### ISSUE THREAD

**Issue by paaljoachim** [fluentform/fluentform#501]
Item Quantity field is not showing up for a possible calculation. If I add a Numeric field instead then that will show up in the calculations area.

Meaning one can not calculate Item Quantity field, but have to use a Numeric field instead.

This is part of a longer thread on Facebook here:
https://www.facebook.com/groups/fluentforms/posts/1817078342434440/?comment_id=1818836735591934&reply_comment_id=1819658648843076

I am working on a tutorial and a few different issues showed up which Shaugat is bringing onward to the developers, but this issue here is new.

**Comment by paaljoachim** [comment]
Additional comment.

Creating a form specific to test the Item Quantity field.
The field showed up so that I was able to calculate.

Going back to the existing form I see I selected to have a Numeric field. I had noticed earlier that Item Quantity Field did not work correctly as it did not show up in the calculations area. Here is an export of the form. I might just have to recreate it as that might help.
https://icedrive.net/s/jjGk15YzgVj4AvNCuvw6aQTaxS4z

Here is a video I made:
https://youtu.be/RAafPkGx8Hg

There seems to be errors with the existing form. I will have to create a new form and see if the Item Quantity shows up.
---> NB. In a new form the Item Quantity field did show up in the calculations area.

Btw
Writing the name of the field field search box the Item Quantity field does not show up. So there seems to be something wrong with the field. As it is not showing up in the field search box or for the above form in the calculations area.
